docs/_docs/reference/metaprogramming/staging.md

```diff
@@ -108,7 +108,7 @@ to get a source-like representation of the expression.
 import scala.quoted.*
 
 // make available the necessary compiler for runtime code generation
-given staging.Compiler = staging.Compiler.make(getClass.getClassLoader)
+given staging.Compiler = staging.Compiler.make(Predef.getClass.getClassLoader)
```

Comment:
That might not be the correct classloader. The standard library might be loaded earlier with a different classloader. We need to get it from a class defined in the current project.

I am not sure what is the cleanest encantation for this one that we cannot the the classloader of the synthetic package object.